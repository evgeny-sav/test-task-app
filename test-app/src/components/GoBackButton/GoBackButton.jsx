import React from 'react';
import { useHistory } from 'react-router-dom';
import { Button } from 'react-bootstrap';

const GoBackButton = () => {
  const { goBack } = useHistory();

  return (
    <Button variant="light" className="mb-5" onClick={() => goBack()}>
      &larr; Back
    </Button>
  );
};

export default GoBackButton;
