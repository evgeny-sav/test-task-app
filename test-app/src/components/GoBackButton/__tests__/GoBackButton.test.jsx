import React from 'react';
import { create } from 'react-test-renderer';
import GoBackButton from '../GoBackButton';

describe('GoBackButton', () => {
  it('should render', () => {
    const component = create(<GoBackButton />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
