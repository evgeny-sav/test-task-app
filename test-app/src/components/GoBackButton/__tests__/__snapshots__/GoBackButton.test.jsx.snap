// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoBackButton renders without crashing 1`] = `
<button
  className="mb-5 btn btn-light"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  ← Back
</button>
`;

exports[`GoBackButton should render 1`] = `
<button
  className="mb-5 btn btn-light"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  ← Back
</button>
`;
